torch/lib/c10d/TCPStore.cpp

```diff
@@ -286,8 +287,10 @@ TCPStore::TCPStore(
     tcpStoreDaemon_ = std::unique_ptr<TCPStoreDaemon>(
         new TCPStoreDaemon(masterListenSocket_));
   }
+  // set the timeout
+  setTimeout(timeout);
   // Connect to the daemon
-  storeSocket_ = tcputil::connect(tcpStoreAddr_, tcpStorePort_);
+  storeSocket_ = tcputil::connect(tcpStoreAddr_, tcpStorePort_, true, timeout_);
```

Comment:
Please add `/* wait= */` before `true` so you can read immediately what it is for. The other args are quite clear except for the wait one. This is useful for opaque params where you cannot infer what's happening by looking at it.